include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -869,20 +881,47 @@ struct SPhysicalDeviceLimits
 
         if (minImportedHostPointerAlignment < _rhs.minImportedHostPointerAlignment) return false;
 
+        if (shaderBufferFloat32AtomicAdd && !_rhs.shaderBufferFloat32AtomicAdd) return false;
+        if (shaderBufferFloat64Atomics && !_rhs.shaderBufferFloat64Atomics) return false;
+        if (shaderBufferFloat64AtomicAdd && !_rhs.shaderBufferFloat64AtomicAdd) return false;
+        if (shaderSharedFloat32AtomicAdd && !_rhs.shaderSharedFloat32AtomicAdd) return false;
+        if (shaderSharedFloat64Atomics && !_rhs.shaderSharedFloat64Atomics) return false;
+        if (shaderSharedFloat64AtomicAdd && !_rhs.shaderSharedFloat64AtomicAdd) return false;
+        if (shaderImageFloat32AtomicAdd && !_rhs.shaderImageFloat32AtomicAdd) return false;
+        if (sparseImageFloat32Atomics && !_rhs.sparseImageFloat32Atomics) return false;
+        if (sparseImageFloat32AtomicAdd && !_rhs.sparseImageFloat32AtomicAdd) return false;
+
+        if (robustStorageBufferAccessSizeAlignment < _rhs.robustStorageBufferAccessSizeAlignment) return false;
+        if (robustUniformBufferAccessSizeAlignment < _rhs.robustUniformBufferAccessSizeAlignment) return false;
```

Comment:
Alignment behaves differently to other limits, because it bounds from below, not above.

For example if device A declares an alignment of 32 and device B declares an alignment of 128, if we have an offset which is a multiple of 64 then device B is uncapable of working with it, hence B's alignment limit is a subset of A's.

**So here you should be comparing for `lhs>rhs` in order to return false.**
(bound from below is greater than right hand side's)

Most limits bound from above, such as max texture size. If device A declares it can only support 8k textures, and device B declares 32k, then A's texture size limit is a subset of B's. So for example creating a 16k texture will be possible on B and A is incapable.